lib/iris/_constraints.py

```diff
@@ -264,11 +264,15 @@ def extract(self, cube):
             desired_values = list(self._coord_thing)
             # A dramatic speedup can be had if we don't have bounds.
             if coord.has_bounds():
-                call_func = lambda cell: cell in desired_values
+                def call_func(cell):
```

Comment:
i find reading over multiple lines clearer than having a def and func on one line, I prefer @QuLogic 's current, I think